docs/migration/4_0.md

```diff
@@ -1,3 +1,6 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+### RouterService
+`RouterService`- Removed 2nd argument `query`. Use `extras.queryParams` instead.
```

Comment:
Let's mention it's method 'go'. Moreover I can see we also changed the protected method `navigate()`: 
```suggestion
`RouterService.go` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
`RouterService.navigate` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
```